packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js

```diff
@@ -391,7 +391,7 @@ test('amm doubleSwap', async t => {
 
   const metricsSub = await E(amm.ammPublicFacet).getMetrics();
   const m = await subscriptionTracker(t, metricsSub);
-  m.assertInitial({ XYK: [] });
+  await m.assertInitial({ XYK: [] });
```

Comment:
> I think not having a marker means that it doesn't wait, and the test will fail if a broken promise isn't handled. Is that closer to what we want in a test? Or am I misunderstanding how the alternatives work?

I think there's some misunderstanding. `void` does not change the handling of the promise; it serves to tell the linter that the non-handling is intentional. 

This code works exactly the same without `void`,
```
test.only('unhandled', async t => {
  const makeRejector = async () => {
    assert.fail('problem');
  };
  void makeRejector();
  t.pass();
});
```
With this Ava output,
```
  Unhandled rejection in test/amm/vpool-xyk-amm/test-xyk-amm-swap.js

  Error: problem
    at makeRejector (packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1174:12)
    at packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1176:3
    at async Promise.all (index 0)

  › makeRejector (packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1174:12)
  › packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1176:3

  ─

  1 test passed [16:12:26]
  1 unhandled rejection
```

I content this code (as above) is better,
```
test.only('unhandled', async t => {
  const makeRejector = async () => {
    assert.fail('problem');
  };
  await makeRejector();
  t.pass();
});
```
because it handles the rejection and pretty-prints the error:
```
  Rejected promise returned by test. Reason:

  Error {
    message: 'problem',
  }

  › makeRejector (packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1174:12)
  › packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js:1176:9

  ─

  1 test failed [16:13:22]
```